### DISCUSSION TITLE
brew install dislocker on macOS Ventura

### DISCUSSION THREAD

**Discussion by superbonaci** [Homebrew/discussions#3844] [Everyday usage]
Would like to install `dislocker` on macOS Ventura to do some testing, but apparently it's disabled:
```
% brew install dislocker
Running `brew update --auto-update`...
==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations

==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).

Error: dislocker has been disabled because it requires closed-source macFUSE!
```
Any possibility to get it working?

**Comment by SMillerDev** [comment]
You can convince macFUSE to open source their project so we can ship it again, beyond that I don't think so.

**Reply by superbonaci** [reply]
What happens if I install the closed source version first then install `dislocker`?

**Reply by superbonaci** [reply]
Is this same macFUSE we are talking about or it another project?
https://github.com/osxfuse/osxfuse

**Reply by SMillerDev** [reply]
That only supports up to 10.14 and if you install the closed source version brew still won't support it.

**Reply by superbonaci** [reply]
Apparently there are several versions of macFUSE, which ones are the open source / Free Software / closed source ones? Could you add such information to README or FAQ? Also the compatibility of each one or lack thereof...

**Comment by gromgit** [comment]
You can try my tap: https://github.com/gromgit/homebrew-fuse/

**Reply by superbonaci** [reply]
Is that 100% equivalent, I mean will work?

**Reply by gromgit** [reply]
I don't use Dislocker, so I can't say whether it'll work for you. What I _can_ say:
1. Everything in my tap was "adopted" from Homebrew core, and modified to deal with build issues like Homebrew's path filtering of `/usr/local` (where macFUSE resides) on ARM64.
2. macFUSE is stuck with the v2 libfuse API, so some formulae cannot be upgraded to the latest version that requires v3.
3. I don't have Ventura installed at the moment, but I have bottles for everything from Mojave to Monterey (except ARM64 Big Sur). I believe Homebrew may automatically select a bottle from an earlier OS; if not, you should be able to build from source in the usual way.

**Reply by superbonaci** [reply]
So in Monterey [homebrew-fuse](https://github.com/gromgit/homebrew-fuse/) will be installed automatically with the command `brew install dislocker`? (In Ventura is not recognized)

**Reply by gromgit** [reply]
No. Please read the README at the URL I posted above, which documents the naming convention I've adopted to avoid namespace clashes with Homebrew core.

**Reply by gromgit** [reply]
Also, you'll have to install the macFUSE cask yourself:
```
brew install --cask macfuse
```

**Reply by superbonaci** [reply]
> Also, you'll have to install the macFUSE cask yourself:
> 
> ```
> brew install --cask macfuse
> ```

Do I need to enable kernel extensions from Startup Security Utility? What to do after installing that macFUSE?

**Reply by gromgit** [reply]
Like I said, I don't have Ventura installed, but macOS itself should prompt you if additional steps are required.

**Comment by superbonaci** [comment]
I've installed `macfuse-4.4.1.dmg`, however I'm still unable to install `dislocker`:
```
$ brew install --ignore-dependencies dislocker
Warning: `--ignore-dependencies` is an unsupported Homebrew developer flag!
Adjust your PATH to put any preferred versions of applications earlier in the
PATH rather than using this unsupported flag!

Error: dislocker has been disabled because it requires closed-source macFUSE!
```
How can I make it work?

**Reply by SMillerDev** [reply]
https://github.com/gromgit/homebrew-fuse/#how-do-i-install-these-formulae
